diff --git a/dist/index.js b/dist/index.js
index 6da0d783c973afa6afe62cab4dc976591d218c12..07c4784c01287a9b0e5d6b6ab1ec7d2c995ad711 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,4 +1,4 @@
-import { fabric as y } from "fabric";
+import * as y from "fabric";
 const x = (C) => Object.keys(C);
 class T {
   aligningLineMargin = 4;
diff --git a/dist/module/aligning.d.ts b/dist/module/aligning.d.ts
index d4718cad2bffda15544db4168dd3df2797d6b849..baab4103c549e33a75653a0a0305bab771edc1e4 100644
--- a/dist/module/aligning.d.ts
+++ b/dist/module/aligning.d.ts
@@ -1,4 +1,4 @@
-import { fabric } from "fabric";
+import * as fabric from "fabric";
 declare type VerticalLineCoords = {
     x: number;
     y1: number;
